### DISCUSSION TITLE
Footnote divider not rendered on GFM based GitHub Pages

### DISCUSSION THREAD

**Discussion by rickstaa** [community/community#49062] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I love that you have given users the ability [to add footnotes to their markdown](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/). However, I noticed that the divider present in GitHub READMEs does not get rendered when the markdown is translated into GitHub pages HTML.

**README**

![image](https://user-images.githubusercontent.com/17570430/222656269-8645b5e9-50fb-4070-9382-879f1207fb80.png)

**GitHub Pages**

![image](https://user-images.githubusercontent.com/17570430/222656363-688ef534-2d13-42d2-b5cd-1a93ee38abdd.png)

## How to reproduce

1. Create [a readme](https://github.com/rickstaa/awesome-adsb/blob/main/readme.md) file that contains the Note/Warning admonitions.
2. Set up GitHub pages according to [GitHub pages documentation](https://docs.github.com/en/pages/quickstart).
3. Use `markdown: GFM` in your Jekyll `_config.yml` file.
4. Build the documentation.
5. [See that the divider before the footnote is not displayed correctly](https://adsb.cool/).

**Comment by ettaboyle** [comment]
Hi again 👋🏻 @rickstaa! You are more likely to get a useful response if you are posting your question(s) in the applicable category. I've gone ahead and moved it from `general` to `pages` for you. Good luck!

**Reply by rickstaa** [reply]
Thanks again 👍🏻!
